resources/js/components/FileCard.vue

```diff
@@ -26,7 +26,18 @@
         >
             <button
                 type="button"
-                v-if="withMeta.canDownload"
+                @keydown.enter.prevent="viewFile"
+                @click.prevent="viewFile"
+                v-tooltip.click="__('View')"
+                v-if="withMeta.canDownload && withMeta.contentDisposition !== 'download'"
```

Comment:
As contentDisposition will be an array/object use this `withMeta.contentDisposition.includes('attachement')`